src/content/docs/accounts/original-accounts-billing/original-product-based-pricing/switch-new-models.mdx

```diff
@@ -1,24 +1,26 @@
 ---
-title: Switch to the new pricing model
+title: Switch to the New Relic One pricing model
 metaDescription: "How to switch to New Relic's newer pricing model."
 redirects: 
   - /docs/accounts/accounts-billing/switch-new-model
 ---
 
-This doc explains how to switch from our original pricing model to the New Relic One pricing model. [Learn more about pricing models.](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-pricing-models)
+Some of our older customers are still on our **original pricing model**. Our newer pricing model, called the **New Relic One pricing model**, is much simpler and offers many benefits compared to our original pricing model ([learn more](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing)). This doc explains the options available for switching to the newer pricing model. 
 
 ## How free accounts switch
 
-Have a free account? You currently have access to all of New Relic One. However, if you’d like to take advantage of the new pricing and user models, please [sign up for a new account](https://newrelic.com/signup/). [Learn what you get for free.](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#free)
+Have a free account on the original pricing model? You currently have access to a wide range of New Relic features. We currently don't have a way to convert free, non-paying customer accounts to the new pricing model. So to get on the newer pricing model, and take advantage of the those benefits, you'll need to [sign up for a new account](https://newrelic.com/signup). 
+
+[Learn what you get for free on the new pricing model.](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing/#free)
 
 ## How paying accounts switch to new pricing [#transition-paid]
 
-If you're a paying account on our [original pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models), you may only switch to New Relic One pricing if one of these is true:
+If your organization is on our [original pricing model](/docs/accounts/original-accounts-billing/original-product-based-pricing/overview-user-models) and you are paying us, you may only switch to New Relic One pricing if one of these is true:
```

Comment:
Maybe:

If your organization pays for data ingestion using our original pricing model, you're eligible to switch to our new pricing model if one of these is true: 

? Could see it being redundant though.